### ISSUE TITLE
Page count index variable

### ISSUE THREAD

**Issue by SwissCheese5** [adamstyrc/calendarro#11]
We can currently get the total number of pages via calendarro.calendarroState.pageCount, but we can't get the current page's index.

**Comment by adamstyrc** [comment]
Well, there could appear some listener for page changes, true. 
For now you can access pageView via:
```dart
    final pageView = calendarro.widget.state.pageView;
    final currentPage = pageView.controller.page
```

**Comment by adamstyrc** [comment]
@SwissCheese5 
I just released Calendarro **1.1.0** where you can pass additional callback **onPageSelected**
```dart
Calendarro(
...
        onPageSelected: (pageStartDate, pageEndDate) {
          //change month label here
        },
)
